invenio_administration/templates/semantic-ui/invenio_administration/edit.html

```diff
@@ -9,7 +9,14 @@
 
 {% block admin_page_content %}
 
+  <div class="mb-10">
+    <a href="{{ request.referrer }}" class="ui small button labeled icon rel-mr-1">
```

Comment:
could you add a check for referrer? Because if referrer is none then the button is broken, while it should not be displayed